### PR TITLE
Update BoilerplateProjects.md

### PR DESCRIPTION
**Pull request by amiya-pattnaik**
updated wdio-v5  boilerplate project for Cucumber

## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by amiya-pattnaik** [comment]
Hi Chris,
Just updated the wdio-v5 boilerplate project for cucumber under v5 and removed the same boilerplate project from v4. Rest of the content remains same. Please review and merge.

Thanks,
Amiya

File: docs/BoilerplateProjects.md [hunk 1]
```diff
@@ -7,48 +7,25 @@ Over time, our community has developed several boilerplate projects that you can
...
-    - Husky
-
-### [WarleyGabriel/demo-webdriverio-cucumber](https://github.com/WarleyGabriel/demo-webdriverio-cucumber)
-
-A powerful boilerplate project to run E2E tests with Cucumber.
+    - [Page Object](PageObjects.md) Models used with ES6 style class-based approach, and full ES6–ES8 support through Babel
+    - Examples of multi selector option to query elements with more than one selector at a time
+    - Examples of headless browser execution, using PhantomJS and Chrome
+    - Integration with BrowserStack, SauceLabs
+    - Support of reading and writing from MS-Excel documentsfor easy test data management from external data sources with examples
```
Comment (baruchvlz) [inline docs/BoilerplateProjects.md]:
```suggestion
    - Support of reading and writing from MS-Excel documents for easy test data management from external data sources with examples
```

File: docs/BoilerplateProjects.md [hunk 2]
```diff
@@ -7,48 +7,25 @@ Over time, our community has developed several boilerplate projects that you can
...
+    - Database support for any RDBMS (Oracle, MySql, TeraData, Vertica etc.), executing queries, fetching result sets, and so on. Includes examples of E2E testing.
+    - Multiple reporting (Spec, Junit, Allure, JSON), plus local view of JUnit report(`.html`) format
+    - Task manage through Grunt
+    - Examples with demo app https://search.yahoo.com  and http://www.phptravels.net, Chai assertion liberary (`expect`, `assert`, `should`)
```
Comment (baruchvlz) [inline docs/BoilerplateProjects.md]:
```suggestion
    - Examples with demo app https://search.yahoo.com and http://www.phptravels.net, Chai assertion liberary (`expect`, `assert`, `should`)
```

File: docs/BoilerplateProjects.md [hunk 3]
```diff
@@ -7,48 +7,25 @@ Over time, our community has developed several boilerplate projects that you can
...
+    - Appium-specific `.config` file for playback on mobile device. For one-click Appium setup, refer to: appium-setup-made-easy-OSX(https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX)
```
Comment (baruchvlz) [inline docs/BoilerplateProjects.md]:
```suggestion
    - Appium-specific `.config` file for playback on mobile device. For one-click Appium setup, refer to: [appium-setup-made-easy-OSX](https://github.com/amiya-pattnaik/appium-setup-made-easy-OSX)
```

File: docs/BoilerplateProjects.md [hunk 4]
```diff
@@ -7,48 +7,25 @@ Over time, our community has developed several boilerplate projects that you can
...
-    - Multiple reports (Spec, Dot, Allure, Timeline)
-    - Visual regression tests (Image comparison service)
-    - Code formatter (ESlint, Prettier)
-    - Husky
-
-### [WarleyGabriel/demo-webdriverio-cucumber](https://github.com/WarleyGabriel/demo-webdriverio-cucumber)
-
-A powerful boilerplate project to run E2E tests with Cucumber.
+    - [Page Object](PageObjects.md) Models used with ES6 style class-based approach, and full ES6–ES8 support through Babel
+    - Examples of multi selector option to query elements with more than one selector at a time
```
Comment (baruchvlz) [inline docs/BoilerplateProjects.md]:
```suggestion
    - Examples of multi-selector option to query elements with more than one selector at a time
```

File: docs/BoilerplateProjects.md [hunk 5]
```diff
@@ -7,48 +7,25 @@ Over time, our community has developed several boilerplate projects that you can
...
-    - Chai
-    - Multiple reports (Spec, Dot, Allure, Timeline)
-    - Visual regression tests (Image comparison service)
-    - Code formatter (ESlint, Prettier)
-    - Husky
-
-### [WarleyGabriel/demo-webdriverio-cucumber](https://github.com/WarleyGabriel/demo-webdriverio-cucumber)
-
-A powerful boilerplate project to run E2E tests with Cucumber.
+    - [Page Object](PageObjects.md) Models used with ES6 style class-based approach, and full ES6–ES8 support through Babel
```
Comment (baruchvlz) [inline docs/BoilerplateProjects.md]:
```suggestion
    - [Page Object](PageObjects.md) Models used with ES6 style class-based approach and full ES6–ES8 support through Babel
```

**Comment by amiya-pattnaik** [comment]
Hi Chris,
Incorporated your comment (removed the wdio-v5 ) 
Please have a look and merge it.
Thanks in advance.
Amiya

**Pull request merged**